src/Codeception/Module/FTP.php

```diff
@@ -25,7 +26,7 @@
  * For SFTP, add [phpseclib](http://phpseclib.sourceforge.net/) to require list.
  * ```
  * "require": {
- *  "phpseclib/phpseclib": "0.3.6"
+ *  "phpseclib/phpseclib": ">=0.3.6"
```

Comment:
It would be better to give an instruction to install  phpseclib 2.*, this module can be incompatible with phpseclib 3.

Reply:
Support for 0.3.6 is for backwards compatibility, there is no reason to tell new users that they can use that old version.
It would be the best to change example to `"phpseclib/phpseclib": "^2.0.14"`

Reply:
done